vtex/faststore
# Migrate Hero Section to App folder

## What's the purpose of this pull request?

Migrate Hero Section to `app folder`. The hero was already migrating, only its dependencies were moved to the app folder, except the Section component that we can migrate after finishing all the sections.
 

## How to test it?
```
Run yarn dev
Make sure that all the current pages are working as expected (/, /office and some /<slug>/p);
Make sure /fs-next-update is rendering with the Hero section without errors.
```
